src/components/dialogs/filters-editor.js

```diff
@@ -144,127 +119,102 @@ const FiltersEditor = ({ filters, onChange }) => {
         });
     }
 
-    return (
-        <div className={classes.root}>
-            <Grid container direction="row" spacing={1}>
-                <Grid item xs={12} sm={3}>
-                    <h3>
-                        <FormattedMessage id="equipmentID" />
-                    </h3>
-                </Grid>
-                <Grid item xs={12} sm={9}>
-                    <CustomTextField
-                        onChange={handleEquipmentID}
-                        value={filters.equipmentID}
-                    />
-                </Grid>
-            </Grid>
+    const nominalVoltageOperators = ['=', '>', '>=', '<', '<='];
 
-            <Grid container direction="row" spacing={1}>
-                <Grid item xs={12} sm={3}>
-                    <h3>
-                        <FormattedMessage id="equipmentName" />
-                    </h3>
-                </Grid>
-                <Grid item xs={12} sm={9}>
-                    <CustomTextField
-                        onChange={handleEquipmentName}
-                        value={filters.equipmentName}
-                    />
-                </Grid>
-            </Grid>
-
-            <Grid container direction="row" spacing={1}>
-                <Grid item xs={9} sm={2}>
-                    <h3>
-                        <FormattedMessage id="nominalVoltage" />
-                    </h3>
-                </Grid>
-                <Grid item xs={3} sm={1}>
-                    <FormControl className={classes.formControl}>
-                        <Select
-                            native
-                            value={filters.nominalVoltageOperator}
-                            onChange={handleOperator}
-                        >
-                            <option value={'='}>=</option>
-                            <option value={'>'}> &gt; </option>
-                            <option value={'>='}> &ge; </option>
-                            <option value={'<'}> &lt; </option>
-                            <option value={'<='}> &le; </option>
-                        </Select>
-                    </FormControl>
-                </Grid>
-                <Grid item xs={12} sm={9}>
-                    <CustomTextField
+    return (
+        <>
+            <CustomTextField
+                margin="dense"
+                label={<FormattedMessage id="equipmentID" />}
+                onChange={handleEquipmentID}
+                value={filters.equipmentID}
+            />
+            <CustomTextField
+                margin="dense"
+                label={<FormattedMessage id="equipmentName" />}
+                onChange={handleEquipmentName}
+                value={filters.equipmentName}
+            />
+            <FormControl fullWidth margin="dense">
+                <InputLabel className={classes.inputLegend}>
+                    <FormattedMessage id="nominalVoltage" />
+                </InputLabel>
+                <Grid>
```

Comment:
You may use Grid Container and Grid Item APIs to avoid customizing CSS width property. I think it's better, but not a big deal.

Reply:
fixed